wren-ai-service/src/pipelines/ask/components/post_processors.py

```diff
@@ -46,6 +46,7 @@ async def run(self, replies: List[str]):
             }
         except Exception as e:
             logger.error(f"Error in GenerationPostProcessor: {e}")
+            logger.exception(e)
```

Comment:
I do believe we could use exception instead of error, no need them both. according to the follow sample, the exception method also print the custom message.

```python
import logging

logging.basicConfig(level=logging.DEBUG)
logger = logging.getLogger(__name__)

try:
    1 / 0
except Exception as e:
    logger.exception(f"Failed to xxx: {e}")

    logger.error(f"Failed to xxx: {e}")

```
and it will look like
```text
ERROR:__main__:Failed to xxx: division by zero <-- **here**
Traceback (most recent call last):
  File "demo.py", line 7, in <module>
    1 / 0
    ~~^~~
ZeroDivisionError: division by zero

ERROR:__main__:Failed to xxx: division by zero
```